src/main/java/org/icgc/argo/workflow_management/gatekeeper/graphql/GatekeeperDataFetchers.java

```diff
@@ -0,0 +1,85 @@
+/*
```

Comment:
some of the GQL files are saying they are new.. but the are just being moved into gatekeeper